docs/examples/howto/add_new_algorithm.rst

```diff
@@ -0,0 +1,174 @@
+Add a new algorithm
+===================
+
+Purpose
+-------------
+
+The main part of ACTS code is located in `Core` packages. 
```

Comment:
Suggest:  The main part of ACTS code is located in the Core packages. For such code to be used in standalone ACTS, an algorithm is needed. Before doing so, the ideas explored in Examples are typically first developed as algorithms. In a second step, the essential parts of this code are then moved to the Core packages, which the algorithm then executes.